net/Socket.cpp

```diff
@@ -698,6 +702,16 @@ int Socket::getPid() const
         return -1;
     }
     return creds.pid;
+#else
+    struct xucred creds;
+    socklen_t credSize = sizeof(struct xucred);
+    if(getsockopt(_fd, 0, LOCAL_PEERCRED, &creds, &credSize) < 0)
+    {
+        LOG_TRC("Failed to get pid via peer creds on " << _fd << ' ' << strerror(errno));
+        return -1;
+    }
+    return creds.cr_pid;
```

Comment:
Condition this to `__FreeBSD__ >= 13`, there was no `cr_pid` before https://reviews.freebsd.org/rS348419 which wasn't MFC'd

Reply:
There is nothing we can do in `<= 12` case, only fail with `#error`, that is, different error message.